pragma/core/client.py

```diff
@@ -39,7 +39,7 @@ def __init__(
     ):
         """
         Client for interacting with Pragma on Starknet.
-        :param network: Target network for the client. Can be a string with URL, one of ``"mainnet"``, ``"testnet"``, ``"local"`` or ``"integration""``
```

Comment:
pragma is live on starknet testnet/mainnet, local network, sharingan and soon on pragma X testnet/mainnet too!

Reply:
Thanks! Put it back in now.